### PR TITLE
Upper Jeuno NPC Cleanup

### PR DESCRIPTION
**Pull request by Xaver-DaRed**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [?] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

Clean NPC scripts, to ease future and inevitable mission and quest conversions.
No logic changes made, except to Paya-Sabya.lua script, that had some forgotten Magicite mission logic.

### CODE REVIEW

File: scripts/zones/Upper_Jeuno/npcs/Ajithaam.lua [hunk 1]
```diff
@@ -45,7 +45,11 @@ Port Jeuno (West to East)
 ]]--
 
 entity.onTrade = function(player, npc, trade)
-    if (trade:getGil() == 300 and trade:getItemCount() == 1 and player:getQuestStatus(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.LURE_OF_THE_WILDCAT) == QUEST_COMPLETED and player:getCurrentMission(TOAU) > xi.mission.id.toau.IMMORTAL_SENTRIES) then
+    if trade:getGil() == 300 and
```
Comment (claywar) [inline scripts/zones/Upper_Jeuno/npcs/Ajithaam.lua]: Can you adjust this conditional to match style guide?
```
if
    <condition>
then
```

File: scripts/zones/Upper_Jeuno/npcs/Guide_Stone.lua [hunk 2]
```diff
@@ -11,7 +11,6 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-
     player:messageSpecial(ID.text.GUIDE_STONE)
```
Comment (claywar) [inline scripts/zones/Upper_Jeuno/npcs/Guide_Stone.lua]: Out of scope, but prime candidate for DefaultActions

**Review by claywar — APPROVED** [review]
Looks reasonable overall, just a couple of additional comments

File: scripts/zones/Upper_Jeuno/npcs/Ajithaam.lua (hunk 1 above)
Comment (Xaver-DaRed) [reply scripts/zones/Upper_Jeuno/npcs/Ajithaam.lua]: oops, missed that.

**Pull request merged**